src/batches/entities/batch.entity.ts

```diff
@@ -0,0 +1,38 @@
+import { Batch, Prisma } from '@prisma/client';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class BatchEntity implements Batch {
```

Comment:
Rename it to something like `BatchResponse`

Reply:
Yes, I will change that